lib/notify.ts

```diff
@@ -0,0 +1,61 @@
+import { NotifyClient } from 'notifications-node-client';
+import { Submission } from 'data/flexibleForms/forms.types';
+import forms from 'data/flexibleForms';
+
+/** send an email notification to someone that a submission needs their approval */
+export const notifyApprover = async (
+  submission: Submission,
+  approverEmail: string,
+  host: string
+): Promise<void> => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY);
+
+  return await notifyClient.sendEmail(
+    process.env.NOTIFY_APPROVER_TEMPLATE_ID,
+    approverEmail,
+    {
+      personalisation: {
+        url: `${host}/people/${submission.residents[0].id}/submissions/${submission.submissionId}`,
+        form_name: forms.find((form) => form.id === submission.formId)?.name,
+        resident_name: submission.residents
+          .map((res) => `${res.firstName} ${res.lastName}`)
+          .join(', '),
+        resident_social_care_id: submission.residents
+          .map((res) => res.id)
+          .join(', '),
+        started_by: submission.createdBy.email,
+      },
+      reference: `${submission.submissionId}-${approverEmail}`,
+    }
+  );
+};
+
+/** send an email notification to a creator that their submission has been rejected */
+export const notifyReturnedForEdits = async (
+  submission: Submission,
+  rejecterEmail: string,
+  host: string,
+  rejectionReason?: string
+): Promise<void> => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY);
```

Comment:
Do Hackney as an organisation have access to Notify and do we have a key set for staging/production?

Reply:
we don't have keys set in production and staging yet. we should do this

Reply:
Do we have approval to get these keys within Hackney? Does seem like something you should have wanted to mention before writing the code.

I don't see how this can be approved until we know we have access to valid Notify keys within Hackney.

Reply:
we've got a live notify team set up that's been through the process already, i can get the keys - we just gotta add them to our environments

Reply:
now added